build.gradle

```diff
@@ -17,7 +17,7 @@ ext {
 }
 
 group 'com.emc.ecs'
-version '1.4'
+version '1.4.0'
```

Comment:
Please update to 1.4.1 since the license has changed and you will need to republish.